telegram/ext/_precheckoutqueryhandler.py

```diff
@@ -43,6 +51,13 @@ async def callback(update: Update, context: CallbackContext)
 
             The return value of the callback is usually ignored except for the special case of
             :class:`telegram.ext.ConversationHandler`.
+        pattern (:obj:`str` | :func:`re.Pattern <re.compile>` | :obj:`callable` | :obj:`type`, \
+            optional):
+            Pattern to test :attr:`telegram.Update.pre_checkout_query.invoice_payload` against. If
```

Comment:
```suggestion
            Pattern to test :attr:`telegram.Update.pre_checkout_query.invoice_payload <telegram.PreCheckoutQuery.invoice_payload>` against. If
```
don't commit these directly from Github, since it will exceed the line limit. Do it in the IDE so you can correct it